src/main/java/taxi/model/Driver.java

```diff
@@ -0,0 +1,36 @@
+package taxi.model;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
+    public Driver(String name, String licenseNumber) {
```

Comment:
check for null everywhere :)

Reply:
I will change everywhere - on next homework)